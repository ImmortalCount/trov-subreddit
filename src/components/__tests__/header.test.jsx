import React from 'react'
import renderer from 'react-test-renderer'
import { Router } from 'react-router-dom'

import history from '../../utils/history'
import Header from '../header'

describe('Header Component', () => {
    it('Header matches snapshot', () => {
        const modalRender = renderer.create(
            <Router history={history}>
                <Header />
            </Router>
        ).toJSON()
        expect(modalRender).toMatchSnapshot()
    })
})
