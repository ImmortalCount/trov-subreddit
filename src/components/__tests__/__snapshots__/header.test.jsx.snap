// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Header matches snapshot 1`] = `
<header
  className="sc-bxivhb kwEEYI"
>
  <a
    className="sc-htpNat heduYG"
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
  <fieldset
    className="sc-bwzfXH eEaCPL"
  >
    <label
      htmlFor="subreddit-select"
    >
      Select a Suggested Subreddit
    </label>
    <select
      className="sc-ifAKCX byVZdX"
      id="subreddit-select"
      onChange={[Function]}
    >
      <optgroup
        label="Select a Suggested Subreddit"
      >
        <option
          value="reactjs"
        >
          reactjs
        </option>
        <option
          value="guitarpedals"
        >
          Guitar Pedals
        </option>
        <option
          value="guitar"
        >
          Guitar
        </option>
        <option
          value="dataisbeautiful"
        >
          dataisbeautiful
        </option>
        <option
          value="DIY"
        >
          DIY
        </option>
        <option
          value="doommetal"
        >
          Doom Metal
        </option>
        <option
          value="javascript"
        >
          javascript
        </option>
        <option
          value="nottheonion"
        >
          Not The Onion
        </option>
      </optgroup>
    </select>
  </fieldset>
  <form
    className="sc-EHOje ieAJSh"
    onSubmit={[Function]}
  >
    <input
      name="subreddit"
      placeholder="Enter a subreddit..."
    />
    <button
      className="sc-bdVaJa hSwCUs"
      color="green"
      onClick={[Function]}
      size="small"
    >
      submit
    </button>
  </form>
</header>
`;
